src/components/Header.js

```diff
@@ -29,6 +30,11 @@ const Header = (props) => {
     const closeBasket = () => {
         setIsBasketOpen(false);
     }
```

Comment:
At the end of each arrow function you should put semicolon(including functional component)